src/lwc/geTemplates/geTemplates.js

```diff
@@ -6,6 +6,9 @@ import cloneFormTemplate from '@salesforce/apex/FORM_ServiceGiftEntry.cloneFormT
 import TemplateBuilderService from 'c/geTemplateBuilderService';
 import { findIndexByProperty } from 'c/utilTemplateBuilder';
 
+// Import custom labels
+import geAssistiveSpinner from '@salesforce/label/c.geAssistiveSpinner';
```

Comment:
Similar to above - would have thought these were included in the geLabelService but I may just be missing something

Reply:
Nope, not missing anything! Updating this.